ctapipe/io/simteleventsource.py

```diff
@@ -816,6 +842,36 @@ def _parse_simulation_header(self):
         )
         return {obs_id: simulation_config}
 
+    def _fill_scheduling_and_observation_blocks(self):
+        """fill scheduling and observation blocks must be run after the
+        simulation config is filled
+        """
+
+        az, alt = self.file_.header["direction"]
+        obs_id = self.obs_ids[0]
+
+        ob_dict = {
+            obs_id: ObservationBlockContainer(
+                obs_id=np.int64(obs_id),
+                producer_id="simulation",
+                state=ObservationBlockState.COMPLETED_SUCCEDED,
+                subarray_pointing_lat=alt * u.deg,
+                subarray_pointing_lon=az * u.deg,
+                subarray_pointing_frame=CoordinateFrameType.ALTAZ,
+            )
+        }
+
+        sb_dict = {
+            obs_id: SchedulingBlockContainer(
+                sb_id=np.int64(obs_id),  # simulations have no SBs, so use the OB id
```

Comment:
dtype

Reply:
The dtype is int64 in the Container definition so it has to be explicitly cast, right?

Reply:
It failed to validate the container if I do not do this.

Reply:
See comments above, the DL0 draft says these should be uint64